### DISCUSSION TITLE
Date data type of format "yyyy-mm-dd"

### DISCUSSION THREAD

**Discussion by rameshkumar70** [pocketbase/pocketbase#4602] [Q&A] [answered]
Hi, 

At the front-end, the HTML input text box of type "date" caters for date values of the format  "yyyy-mm-dd". In pocketbase, I noticed a DateTime field caters for date/time values which follow the format of  "yyyy-mm-dd hh:mm:ssTZ".  I could certainly read off pocketbase and truncate the "time part" to get the "yyyy-mm-dd" part for easier comparison operations at the front-end. Is there a strictly "date" type of field I could pick in pocketbase which caters for "yyyy-mm-dd" only? I couldn't find anything from the docs. 
Many thanks!

**Comment by ganigeorgiev** [comment, marked as answer]
No, we don't have such field at the moment (_it exists in my todo to consider adding a custom date format/layout field option but for now remains a low priority; I'll reevaluate it during the schema fields restructuring_).

**Comment by rameshkumar70** [comment]
Thank you sir

**Reply by stamler** [reply]
@rameshkumar70  I'm currently working through this. I think my approach will be to specify a string column in pocketbase with a regex to restrict it to YYYY-MM-DD then use a hook to validate that the specified value is actually a real date using time.Parse() in go.
